control_plane/src/background_process.rs

```diff
@@ -60,6 +61,7 @@ pub async fn start_process<F, Fut, AI, A, EI>(
     envs: EI,
     initial_pid_file: InitialPidFile,
     process_status_check: F,
+    retry_timeout_in_seconds: Option<u64>,
 ) -> anyhow::Result<()>
```

Comment:
1. Put the arg before the `process_status_check` closure. Call sites are easier to read that way (closure comes last).

2. All call sites use `Some()`, no need for an `Option<>`.

Reply:
done